js/sketch.js

```diff
@@ -64,14 +66,14 @@ let cowBGMStart = false;
 
 // Player inventory - increases when harvesting and decreases when cooking
 let inventory = {
```

Comment:
going to leave these as 10 as part of development because growing is taking a while. made a note to change before final submission